queue-check.ps1

```diff
@@ -9,42 +9,42 @@ param (
 )
 
 function Invoke-Script() {
-  $WowProcessId = (Get-Process -name "WowClassic").id
-
-  if ($null -eq $WowProcessId) {
-    Write-Host "Wow classic not found. Is it running?"
-    return
-  }
-  
-  Write-Host "Found wow process: $WowProcessId"
-
-  $WowSockets = Get-NetTcpConnection -OwningProcess $WowProcessId
-
-  if ($debug) { Out-Host -InputObject $WowSockets }
+  $WowProcessIds = (Get-Process -name "WowClassic").id
   
-  if (Test-TcpConnectionIsEstablished $WowSockets $QueuePort) {
-    Write-Host "Found connection to queue port $QueuePort. Assuming user is in queue"
-    Send-Discord("$DiscordUserName is in queue")
-    
-    return
+  if ($null -eq $WowProcessIds) {
+  	Write-Host "Wow classic not found. Is it running?"
+	return
   }
   
-  if (Test-TcpConnectionIsEstablished $WowSockets $InGamePort) {
-    Write-Host "Found connection to server port $InGamePort. Assuming user is playing on a server"
-    Send-Discord("$DiscordUserName is playing")
-   
-    return
-  }
+  Write-Host "Found wow processes: $WowProcessIds"
+  
+  Foreach($WowProcessId in $WowProcessIds){
+	
+	Write-Host "Checking: $WowProcessId"
 
-  if (Test-TcpConnectionIsEstablished $WowSockets $LobbyPort) {
-    Write-Host "Found connection to lobby port $LobbyPort. Assuming user is in lobby"
-    Send-Discord("Warning: <@$DiscordUserId> is in lobby")
-    
-    return
+	$WowSockets = Get-NetTcpConnection -OwningProcess $WowProcessId
+
+	if ($Debug) { Out-Host -InputObject $WowSockets }
+	
```

Comment:
LobbyPort could maybe be renamed to something else, since it's active in 3 different scenarios:
1. In queue
2. On character selection screen
3. In game